kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/LibraryChannelCard.vue

```diff
@@ -0,0 +1,115 @@
+<template>
```

Comment:
I think rather than making a new component here, we should use the existing `ChannelCard` component, and add a prop that defaults to false as to whether to show the version details. I double checked with @jtamiace, and this was the intention of the underspecced channel cards in the Figma.

Reply:
Ahh I see. Thanks for the clarification.

Reply:
Changes have been made as requested